README.md

```diff
@@ -1,172 +1,244 @@
-virtualmachinepreference.instancetype.kubevirt.io/ubuntu.22.04 unchanged
-virtualmachinepreference.instancetype.kubevirt.io/windows.10 unchanged
-virtualmachinepreference.instancetype.kubevirt.io/windows.10.virtio unchanged
-virtualmachinepreference.instancetype.kubevirt.io/windows.11 unchanged
-virtualmachinepreference.instancetype.kubevirt.io/windows.11.virtio unchanged
-virtualmachinepreference.instancetype.kubevirt.io/windows.2k12 unchanged
-virtualmachinepreference.instancetype.kubevirt.io/windows.2k12.virtio unchanged
-virtualmachinepreference.instancetype.kubevirt.io/windows.2k16 unchanged
-virtualmachinepreference.instancetype.kubevirt.io/windows.2k16.virtio unchanged
-virtualmachinepreference.instancetype.kubevirt.io/windows.2k19 unchanged
-virtualmachinepreference.instancetype.kubevirt.io/windows.2k19.virtio unchanged
-```
-## Requirements
+* `make schema` (optional)
+  * [openapi2jsonschema](https://github.com/instrumenta/openapi2jsonschema)
 
-- kustomize
-- yamllint
+## Resources
 
-## Build
+The following instance type and preference resources are provided by this project:
 
 ```
-$ make
+
+VirtualMachineClusterInstancetype
+highperformance.large
+---
+VirtualMachineClusterInstancetype
+highperformance.medium
+---
+VirtualMachineClusterInstancetype
+highperformance.small
+---
+VirtualMachineClusterInstancetype
+server.large
+---
+VirtualMachineClusterInstancetype
+server.medium
+---
+VirtualMachineClusterInstancetype
+server.small
+---
+VirtualMachineClusterInstancetype
+server.tiny
+---
+VirtualMachineClusterPreference
+alpine
+---
+VirtualMachineClusterPreference
+centos.7
+---
+VirtualMachineClusterPreference
+centos.7.desktop
+---
+VirtualMachineClusterPreference
+centos.8
+---
+VirtualMachineClusterPreference
+centos.8.desktop
+---
+VirtualMachineClusterPreference
+centos.9
+---
+VirtualMachineClusterPreference
+centos.9.desktop
+---
+VirtualMachineClusterPreference
+cirros
+---
+VirtualMachineClusterPreference
+fedora.35
+---
+VirtualMachineClusterPreference
+fedora.36
+---
+VirtualMachineClusterPreference
+rhel.7
+---
+VirtualMachineClusterPreference
+rhel.7.desktop
+---
+VirtualMachineClusterPreference
+rhel.8
+---
+VirtualMachineClusterPreference
+rhel.8.desktop
+---
+VirtualMachineClusterPreference
+rhel.9
+---
+VirtualMachineClusterPreference
+rhel.9.desktop
+---
+VirtualMachineClusterPreference
+ubuntu.18.04
+---
+VirtualMachineClusterPreference
+ubuntu.20.04
+---
+VirtualMachineClusterPreference
+ubuntu.22.04
+---
+VirtualMachineClusterPreference
+windows.10
+---
+VirtualMachineClusterPreference
+windows.10.virtio
+---
+VirtualMachineClusterPreference
+windows.11
+---
+VirtualMachineClusterPreference
+windows.11.virtio
+---
+VirtualMachineClusterPreference
+windows.2k12
+---
+VirtualMachineClusterPreference
+windows.2k12.virtio
+---
+VirtualMachineClusterPreference
+windows.2k16
+---
+VirtualMachineClusterPreference
+windows.2k16.virtio
+---
+VirtualMachineClusterPreference
+windows.2k19
```

Comment:
Its missing win2k22

Reply:
ACK i'll add this in a separate PR.

Reply:
https://github.com/kubevirt/common-instancetypes/pull/4